index.bs

```diff
@@ -387,233 +507,172 @@ Each \[[Stage]] value indicates the currently pending operation. If the \[[Resul
-  </tr>
-  <tr>
-    <td>\[[Pipeline]]</td>
-    <td>A List</td>
-    <td>A list whose elements are stage records.</td>
-  </tr>
-  <tr>
-    <td>\[[Metadata]]</td>
-    <td>Object or <code>undefined</code></td>
-    <td>The metadata object passed through the pipeline.</td>
-  </tr>
-  <tr>
-    <td>\[[Dependencies]]</td>
-    <td>List of pairs of String and (<a>registry entry</a> or <code>undefined</code>), or <code>undefined</code>.</td>
-    <td>Table mapping unresolved names to their resolved modules.</td>
-  </tr>
-  <tr>
-    <td>\[[Module]]</td>
-    <td>Module Record or Function object or <code>undefined</code></td>
-    <td>The Module Record if the entry is ready, or a thunk if the entry is delayed; otherwise <code>undefined</code>.</td>
-  </tr>
-  <tr>
-    <td>\[[Error]]</td>
-    <td>Any or <b>nothing</b></td>
-    <td>An error that was encountered during one of the phases of the loading pipeline; <b>nothing</b> if no error has been encountered.</td>
-  </tr>
-</table>
-
-<h3 id="registry-constructor">The Registry Constructor</h3>
-
-The Registry constructor is the %Registry% intrinsic object. When called as a constructor it creates and initializes a new %Registry% object. %Registry% is not intended to be called as a function and will throw an exception when called in that manner.
-
-<h4 id="new-registry" aoid="Registry">Registry(loader)</h4>
+<h4 id="new-registry-entry" aoid="RegistryEntry">RegistryEntry(loader)</h4>
 
 When Registry is called with argument <i>loader</i>, the following steps are taken:
 
 <emu-alg>
 1. If NewTarget is *undefined*, then throw a *TypeError* exception.
 1. If Type(_loader_) is not Object, throw a *TypeError* exception.
-1. Let _O_ be OrdinaryCreateFromConstructor(NewTarget, "%RegistryPrototype%", «[[RegistryData]]» ).
+1. Let _O_ be OrdinaryCreateFromConstructor(NewTarget, "%RegistryPrototype%", «[[Loader]], [[Pipeline]], [[Module]], [[Metadata]], [[Dependencies]], [[Error]]» ).
 1. ReturnIfAbrupt(_O_).
-1. Set _O_’s [[RegistryData]] internal slot to a new empty List.
+1. Let _pipeline_ be a new List.
+1. Add new stage entry record { [[Stage]]: "fetch", [[Result]]: *undefined* } as a new element of the list _pipeline_.
+1. Add new stage entry record { [[Stage]]: "translate", [[Result]]: *undefined* } as a new element of the list _pipeline_.
+1. Add new stage entry record { [[Stage]]: "instantiate", [[Result]]: *undefined* } as a new element of the list _pipeline_.
+1. Add new stage entry record { [[Stage]]: "satisfy", [[Result]]: *undefined* } as a new element of the list _pipeline_.
+1. Add new stage entry record { [[Stage]]: "link", [[Result]]: *undefined* } as a new element of the list _pipeline_.
+1. Add new stage entry record { [[Stage]]: "ready", [[Result]]: *undefined* } as a new element of the list _pipeline_.
 1. Set _O_’s [[Loader]] internal slot to _loader_.
+1. Set _O_’s [[Pipeline]] internal slot to _pipeline_.
+1. Set _O_’s [[Module]] internal slot to *undefined*.
```

Comment:
I was following the existing definition for that slot: 

```
<tr>
    <td>\[[Module]]</td>
    <td>Module Record or Function object or <code>undefined</code></td>
    <td>The Module Record if the entry is ready, or a thunk if the entry is delayed; otherwise <code>undefined</code>.</td>
  </tr>
```